### PR TITLE
test(crux-ui): add e2e tests to node screen

### PR DESCRIPTION
**Pull request by balanceee**

### CODE REVIEW

File: web/crux-ui/e2e/nodes.spec.ts [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+  await expect(curl).toBeVisible()
+  await expect(curl).toHaveValue(/curl/)
+  await expect(script).toBeVisible()
+  await expect(timer).toBeVisible()
+  await expect(discardButton).toBeVisible()
+  await expect(copyButton).toBeVisible()
+})
+
+test('Install dagent should be successful', async ({ page }) => {
```
Comment (polaroi8d) [inline web/crux-ui/e2e/nodes.spec.ts:82]: Wow!!! 🔥

**Review by polaroi8d — APPROVED** [review]
Awesome job, small quesiton but looks good to me.

File: web/crux-ui/e2e/nodes.spec.ts [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+  const commandInput = await page.locator('input[readonly]')
+  const curl = await commandInput.inputValue()
+
+  const settings: ExecOptions =
+    process.platform === 'win32'
+      ? {
+          shell: 'C:\\Program Files\\git\\git-bash.exe',
+        }
+      : null
```
Comment (polaroi8d) [inline web/crux-ui/e2e/nodes.spec.ts:113]: What if I run the test in MacOS or on the Github Actions?

File: web/crux-ui/e2e/nodes.spec.ts [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+
+  exec(curl, settings, (err, stdout, stderr) => {
```
Comment (robot9706) [inline web/crux-ui/e2e/nodes.spec.ts:115]: I think the test should remove the dagent container after the test is finished.

**Review by robot9706 — CHANGES_REQUESTED** [review]

**Review by robot9706 — APPROVED** [review]

**Pull request merged**